packages/react-events/src/dom/Press.js

```diff
@@ -680,6 +680,8 @@ const PressResponder: ReactDOMEventResponder = {
       case 'mousedown':
       case 'touchstart': {
         if (!isPressed) {
+          state.shouldPreventClick = false;
```

Comment:
We want to move this down, otherwise we'll set this during simulated events too (which we didn't do before).